doc-content/jbpm-docs/src/main/asciidoc/BAM/DashbuilderRuntime-exporting-importing-dashbuilder-data-proc.adoc

```diff
@@ -0,0 +1,193 @@
+[id='DashbuilderRuntime-exporting-importing-dashbuilder-data-proc_{context}']
+= Exporting and importing Dashbuilder data
+
+ifdef::PAM,JBPM[]
+
+Dashbuilder is a dashboard and reporting tool integrated in {CENTRAL} and is used by the Datasets editor and Content Manager page. There are three data types:
+
+* Datasets
+* Pages
+* Navigation
+
+You can import and export Dashbuilder data as ZIP files in {CENTRAL}.
+
+IMPORTANT: This feature is only accessible by administrator users.
+
+== Exporting Dashbuilder data
+
+You can export the dashbuilder related data such as datasets, pages, and navigation from {CENTRAL} as a ZIP file.
```

Comment:
Sometimes you have it cap "Dashbuilder", others like here lowercase "dashbuilder". Suggest caps throughout for consistency. Suggest a quick search and update as needed. Also, I guess technically it should be "Dashbuilder-related data" hyphenated since a adjective phrase, but honestly, probably not even needed, just fine as "Dashbuilder data".

Reply:
Changed to cap "Dashbuilder".